src/lib/mcp-health-monitor.ts

```
@@ -0,0 +1,256 @@
+
+    const startTime = Date.now();
+    let isHealthy = false;
+    let error: string | undefined;
+
+    try {
+      switch (server.type) {
+        case 'http':
+        case 'sse':
+          isHealthy = await this.checkHttpServer(server);
+          break;
+        case 'stdio':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)